about.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/about.css">
+    <!--  Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=G-ZYTLN5HPEC"></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date());
+    
+      gtag('config', 'G-ZYTLN5HPEC');
+    </script>
+    <title>CIO - CTO Global Summit</title>
+</head>
+<body>
+    <header class="top-header">
+        <div class="header-content">
+            <ul class="header-list">
+                <li><img src="images/facebook.png" alt="Facebook"></li>
+                <li><img src="images/twitter.png" alt="twitter"></li>
+                <li>English</li>
+                <li>My Page</li>
+                <li>Logout</li>
+            </ul>
+        </div>
+    </header>
+
```

Comment:
- [ ] Looks like content is overlapping on the about page also.
![Screenshot from 2021-08-14 01-47-54](https://user-images.githubusercontent.com/1642014/129425053-247f96b1-9548-4743-9143-213d711977d3.png)

Reply:
✔️ The overlapping issue has being fixed for this section